sdk/search/search/README.md

```diff
@@ -78,7 +78,19 @@ const client = new SearchIndexClient(
 
 ### SearchIndexClient
 
-`SearchIndexClient` is one of the primary interface for developers using the Cognitive Search client library. It provides asynchronous methods for working with documents in an index.
+`SearchIndexClient` is one of the primary interface for developers using the Cognitive Search client library. It provides asynchronous methods for working with documents in an index. These methods allow you to query, upload, update, and delete documents. It also provides methods for building completion and suggestion experiences based on partial queries.
+
+### Pagination
+
+Typically you will only wish to [show a subset of search results](https://docs.microsoft.com/en-us/azure/search/search-pagination-page-layout#total-hits-and-page-counts) to a user at one time. To support this, you can use the `top`, `skip` and `includeTotalResultCount` parameters to provide a paged experience on top of search results.
```

Comment:
nit: When linking to docs, remove `en-us/` from the path so the reader's browser locale can direct them to a localized page.